src/stories/Dropdown.stories.js

```diff
@@ -0,0 +1,15 @@
+import { html } from "lit";
+
+export const Default = () => {
+  return html` <dp-dropdown></dp-dropdown>`;
```

Comment:
Se podría hacer así? de esta manera los cambios en AngularJS serían mínimos, no?

```suggestion
  return html` <dp-dropdown>
    <li>Hola soy un li</li>
    <li>Hola soy un li</li>
    <li>Hola soy un li</li>
    <li>Hola soy un li</li>
    <li>Hola soy un li</li>
  </dp-dropdown>`;
```

Reply:
Si se puede, estará implementado en el próximo update de la POC.

Reply:
Ya se encuentra implementado, se puede agregar cualquier elemento.